utils/mysql.cnf

```diff
@@ -0,0 +1,46 @@
+# For advice on how to change settings please see
+# http://dev.mysql.com/doc/refman/5.7/en/server-configuration-defaults.html
```

Comment:
Is the initial template of this file copied from somewhere? If yes, maybe add a direct link to that too (it was not obvious to me following the above link).

Reply:
I should probably remove this file for now and use the default config!

Reply:
I have provided the link to the source as well as brief details